pallets/kensetsu/src/lib.rs

```diff
@@ -676,13 +687,34 @@ pub mod pallet {
             Ok(())
         }
 
+        /// Updates the borrow tax applied during borrow.
+        ///
+        /// ## Parameters
+        ///
+        /// - `origin`: The origin of the transaction.
+        /// - `new_borrow_tax`: The new borrow tax percentage to be set.
+        #[pallet::call_index(9)]
+        #[pallet::weight(<T as Config>::WeightInfo::update_borrow_tax())]
+        pub fn update_borrow_tax(origin: OriginFor<T>, new_borrow_tax: Percent) -> DispatchResult {
+            let who = ensure_signed(origin)?;
+            Self::ensure_risk_manager(&who)?;
+            BorrowTax::<T>::mutate(|borrow_tax| {
+                *borrow_tax = new_borrow_tax;
```

Comment:
It could be [0, 100].

Reply:
Add tests for them

Reply:
0 is default value and it is tested in every test, for example successful path in `test_borrow_sunny_day`. 100% has the same execution path as 1%. Technically `BorrowTax` could be any number, because it is added to the debt, but for sanity it is limited by Percent type which has 100% as max. So, I see no reason to add more tests because they will test already covered paths and will be redundant.

Reply:
Hm, ok. It seemed to me it is possible to set the debt = 0 by tax = 100%. My bad